Sources/PayPalNativeCheckout/PayPalClient.swift

```diff
@@ -13,60 +9,67 @@ public class PayPalClient {
 
     let config: CoreConfig
 
-    // swiftlint:disable identifier_name
-    let CheckoutFlow: CheckoutProtocol.Type
-    // swiftlint:enable identifier_name
+    let nativeCheckout: CheckoutProtocol
 
     /// Initialize a PayPalClient to process PayPal transaction
     /// - Parameters:
     ///   - config: The CoreConfig object
     ///   - returnURL: The return URL provided to the PayPal Native UI experience. Used as part of the authentication process to identify your application. This value should match the one set in the `Return URLs` section of your application's dashboard on your [PayPal developer account](https://developer.paypal.com)
     public init(config: CoreConfig) {
         self.config = config
-        self.CheckoutFlow = Checkout.self
+        let nxoConfig = CheckoutConfig(
+            clientID: config.clientID, createOrder: nil, onApprove: nil,
+                    onShippingChange: nil, onCancel: nil, onError: nil, environment: config.environment.toNativeCheckoutSDKEnvironment())
+        self.nativeCheckout = NativeCheckout(nxoConfig: nxoConfig)
     }
 
-    init(config: CoreConfig, checkoutFlow: CheckoutProtocol.Type) {
+    init(config: CoreConfig, checkoutFlow: CheckoutProtocol) {
         self.config = config
-        self.CheckoutFlow = checkoutFlow
+        self.nativeCheckout = checkoutFlow
     }
 
     /// Present PayPal Paysheet and start a PayPal transaction
     /// - Parameters:
     ///   - request: the PayPalRequest for the transaction
     ///   - presentingViewController: the ViewController to present PayPalPaysheet on, if not provided, the Paysheet will be presented on your top-most ViewController
     ///   - completion: Completion block to handle buyer's approval, cancellation, and error.
-    public func start(request: PayPalRequest, presentingViewController: UIViewController? = nil) {
-        CheckoutFlow.set(config: config)
-
-        CheckoutFlow.start(
-            presentingViewController: presentingViewController,
-            createOrder: { order in
-                order.set(orderId: request.orderID)
-            },
-            onApprove: { approval in
-                self.notifySuccess(for: approval)
-            },
-            onCancel: {
-                self.notifyCancellation()
-            },
-            onError: { errorInfo in
-                self.notifyFailure(with: errorInfo)
-            }
-        )
+    public func start(presentingViewController: UIViewController?, orderID: String, deleagate: PayPalDelegate?){
```

Comment:
typo in the `delegate` parameter here. Currently says `deleagate`. Need one additional space after the end between the `)` and '{' as well.